src/main/telemetry/crsf.c

```diff
@@ -238,6 +280,32 @@ typedef enum {
 static uint8_t crsfScheduleCount;
 static uint8_t crsfSchedule[CRSF_SCHEDULE_COUNT_MAX];
 
+void scheduleMspResponse() {
+    if (!mspReplyPending) {
+        mspReplyPending = true;
+    }
+}
+
+void crsfSendMspResponse(uint8_t *packet) 
+{
+    sbuf_t crsfPayloadBuf;
+    sbuf_t mspPayload;
+    sbuf_t *dst = &crsfPayloadBuf;
+    sbuf_t *msp = &mspPayload;
+
+    msp->ptr = packet;
+    msp->end = packet + CRSF_FRAME_TX_MSP_PAYLOAD_SIZE;
```

Comment:
Use `sbuf_t *msp = sbufInit(&mspPayload, packet, packet + CRSF_FRAME_TX_MSP_PAYLOAD_SIZE);`

Reply:
Won't using the CRSF frame size couple this code directly with CRSF?  It was intended to be generic to any protocol.